### PR TITLE
Choose Starting Player As X or O 

### PR DESCRIPTION
**Pull request by advait2205**
Here feature to confirm player before start of every game is added as shown below
final1_SparkVideo

Fixes: #6

### CODE REVIEW

File: app/src/main/java/com/adit/tictactoe/MainActivity.java
Comment (adit19shah) [inline app/src/main/java/com/adit/tictactoe/MainActivity.java]: Please maintain the indent that was there initially.

Comment (advait2205) [reply app/src/main/java/com/adit/tictactoe/MainActivity.java]: Actually what are you seeing is a outdated version it was then changed because when i made a pull request another PR was merged which is for changing array declaration from C style to Java. SO, i had a "merge to upstream resolve" and i changed it to earlier indentation

Comment (advait2205) [reply app/src/main/java/com/adit/tictactoe/MainActivity.java]: Allthough it was outdated i made sure in recent PR #17 that earlier indentation is maintained

**Review by vidhibadrakiya — APPROVED** [review]

**Review by adit19shah — APPROVED** [review]

**Comment by adit19shah** [comment]
LGTM, Thanks. Will merge once @sarthakhp approves it.

**Comment by advait2205** [comment]
> LGTM, Thanks. Will merge once @sarthakhp approves it.

@sarthakhp Has requested changes in PR #17 which is actually the change in PR #16 so should i do changes in this PR?

**Comment by adit19shah** [comment]
> > LGTM, Thanks. Will merge once @sarthakhp approves it.
> 
> @sarthakhp Has requested changes in PR #17 which is actually the change in PR #16 so should i do changes in this PR?

Yes, you need to make that change in this PR, PR #16

**Comment by advait2205** [comment]
> > > LGTM, Thanks. Will merge once @sarthakhp approves it.
> > 
> > 
> > @sarthakhp Has requested changes in PR #17 which is actually the change in PR #16 so should i do changes in this PR?
> 
> Yes, you need to make that change in this PR, PR #16

Required Changes Done as below : 

Screenshot 2021-10-05 122326

**Review by sarthakhp — APPROVED** [review]

**Pull request merged**
